### DISCUSSION TITLE
Use "gzip-size" instead of "gzip"

### DISCUSSION THREAD

**Discussion by bsastregx** [lit/lit#3854] [Ideas]
[Lit Typescript starter](https://github.com/lit/lit-element-starter-ts) comes with the following script in package.json: 

```
"checksize": "rollup -c ; cat my-element.bundled.js | gzip -9 | wc -c ; rm my-element.bundled.js",
```
it uses **gzip**, which from what i heard, is available by default for MacOs and most flavors of Linux, but not in Windows, which is widely used.

If you try to run that script on Windows you will get:
```
"gzip" is not recognized as an internal or external command.
```

`gzip-size` does the same, and it comes already installed since it is a dependency of `rollup-plugin-summary`. Why don't just add `gzip-size-cli` as a dependency, and then just use that instead of `gzip` ?

**Comment by justinfagnani** [comment]
We could do that. We could also use `rollup-plugin-summary` like we do for the core libraries.
